IncExpTrack/MVVM/Views/StatisticsPage.xaml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:charts="clr-namespace:Syncfusion.Maui.Charts;assembly=Syncfusion.Maui.Charts"
+             x:Class="IncExpTrack.MVVM.Views.StatisticsPage"
+             Title="StatisticsPage">
+
+    <Grid RowDefinitions=".15*, .15*, .30*, .40*">
```

Comment:
Of course planning to add the graphs here looks good

Reply:
That's the plan of course